scripts/start.js

```diff
@@ -4,12 +4,12 @@ const webpackConfig = require('../config/webpack/webpack.config');
 const builds = require('../config/builds');
 const opn = require('opn');
 
-const { hosts, port } = builds[0];
+const { hosts, port, historyApiFallback } = builds[0];
 
 const compiler = webpack(webpackConfig);
 const devServer = new WebpackDevServer(compiler, {
   contentBase: builds.map(({ paths }) => paths.public),
-  historyApiFallback: true,
+  historyApiFallback: historyApiFallback ? historyApiFallback : true,
```

Comment:
Not sure that the logic is correct here. What are the expected values for historyApiFallback?

Reply:
The usecase may be dependant on how you serve files in production. I'm away from desk but can try and hand write an example:
`{rewrite: {from: /./, to: '404'}}`
It would be good to agree on a pattern rather than allowing custom config, but not sure what Configs there would be.